tests/tests_suite_test.go

```
@@ -104,7 +105,12 @@ var _ = SynchronizedBeforeSuite(testsuite.SynchronizedBeforeTestSetup, testsuite
 var _ = SynchronizedAfterSuite(testsuite.AfterTestSuiteCleanup, testsuite.SynchronizedAfterTestSuiteCleanup)
 
 var _ = AfterEach(func() {
-	testCleanup()
+	skipGlobalCleanupLabelText := decorators.SkipGlobalCleanup[0]
+
+	skipGlobalCleanup, err := CurrentSpecReport().MatchesLabelFilter(skipGlobalCleanupLabelText)
+	if !skipGlobalCleanup || err != nil {
```

I would expect an error to not occur here, not silently proceed

---

`Expect(err).ToNot(HaveOccurred())` should be fine

---

You're in an aftereach, should be fine

---

This aftereach is no different to any other one found in any test suite

---

Anyway you can just panic() if you want it's not a big deal, just no reason for this to ever error